src/main/java/pl/jakubkozlowski/learning/firststeps/dto/UserDTO.java

```diff
@@ -0,0 +1,24 @@
+package pl.jakubkozlowski.learning.firststeps.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class UserDTO {
+    private Long id;
+    private String username;
+    private String email;
+    private String password;
+    private Integer favoriteRoleId;
```

Comment:
could think of using enum for that matter. Just pointing for the future reference